Client/Frontend/Browser/Browser.swift

```diff
@@ -13,13 +13,14 @@ protocol BrowserHelper {
 }
 
 class Browser: NSObject, WKScriptMessageHandler {
-    let webView = WKWebView(frame: CGRectZero, configuration: WKWebViewConfiguration())
-
-    override init() {
-        super.init()
+    let webView: WKWebView
 
+    init(configuration: WKWebViewConfiguration) {
+        configuration.userContentController = WKUserContentController()
+        webView = WKWebView(frame: CGRectZero, configuration: configuration)
         webView.allowsBackForwardNavigationGestures = true
-        webView.configuration.userContentController = WKUserContentController()
```

Comment:
I saw your note about the order of creating/initializing things being important. Are you sure this did not work before? Because I am successfully adding user scripts to the `webView.configuration.userContentController` from the ReaderMode helper without issues.